Gemfile

```diff
@@ -0,0 +1 @@
+gem 'aws-idempotency', '0.0.1', :git => 'git@github.com:cyanna/aws-idempotency.git'
```

Comment:
We should be able to use the :github => 'cyanna/aws-idempotency' shorthand here.